helpers/query_helper.py

```diff
@@ -55,15 +59,18 @@
 
 
 def build_new_patrons_query(creation_dt_start):
-    return _NEW_PATRONS_QUERY.format(cached_creation_dt=creation_dt_start,
-                                     limit=os.environ['SIERRA_BATCH_SIZE'])
+    return _NEW_PATRONS_QUERY.format(
+        cached_creation_dt=creation_dt_start,
+        limit=os.environ['SIERRA_BATCH_SIZE'],
+        total_limit=int(os.environ['SIERRA_BATCH_SIZE']) * 2)
```

Comment:
How do `limit` and `total_limit` work together? I'm unclear on why it would be double the value

Reply:
limit acts on the record_metadata table and is essentially the 'real' limit that we're aiming for. However, a small number of duplicate records (~5%) are generated from the JOIN so that if 'limit' is 100, there will be about 105 records returned by the full query. It would probably be fine to not include total_limit at all but just for safety purposes I included one and set it to be large enough that it should never actually be hit (unless something is wrong or drastically changes).